### PR TITLE
Added support for social icons in footer. Resolves #53

### LINKED ISSUE #53: Social media icons
At present we lack easy navigation to official Rebble social media and related outlets - it would be incredibly handy if these were easily reachable from the site:

* [Twitter](https://twitter.com/pebble_dev)
* [Facebook](https://www.facebook.com/RebbleAlliance/)
* [Discord](https://discordapp.com/invite/aRUAYFN)
* [Mailing list](https://ishotjr.us14.list-manage.com/subscribe/post?u=602157eb0bbaa9eb99f18f82a&id=1651ac902e)

*@potatofi I'm perhaps looking at you if you want to create really cool Rebble-y social media icons for us...!* 💖

### PR DESCRIPTION
**Pull request by Willow-Systems**
Added pretty icons! :framed_picture: 

Adds PotatoFi's icons from #53 in the footer of every page, under the 'unofficially official' text.

Thoughts on the icon placement / size?

Pic:
image

### CODE REVIEW

**Comment by ishotjr** [comment]
@Willow-Systems amazing - thanks for taking this! 😍 Do we still need the text Discord link if we have the icon? How might it look with the Subscribe input on the same line as the icons? ❤️

**Comment by Willow-Systems** [comment]
I think everything on one line would look too busy, but here it is without the text:

image

Not sure it's as inviting! :grin:

**Comment by ishotjr** [comment]
@Willow-Systems I'm going to pull your changes locally and have a play... :bowtie:

**Comment by ishotjr** [comment]
@Willow-Systems OK, yeah, I tried it inline and it is indeed too busy! And I agree RE keeping the text in order to really highlight the Discord community! :bowtie: :+1:

File: _includes/footer.html [hunk 1]
```diff
@@ -3,5 +3,10 @@
         <p>Join us on the <a href="https://discordapp.com/invite/aRUAYFN">
             officially unofficial Pebble Discord server!</a></p>
     </div>
+    <div class="text-content">
+      {% for item in site.data.socials %}
```
Comment (ishotjr) [inline _includes/footer.html:7]: love the YML-driven approach vs. hard-coded! :man_cook: :ok_hand:

**Pull request merged**